### PR TITLE
Set the `LANGUAGE` environment variable in the Docker backend

### PR DESCRIPTION
**Pull request by stevepiercy**
I also overhauled the multilingual docs and added a new section for the usage of the change in the Makefile.

See https://github.com/plone/volto/issues/6219#issuecomment-2266129769.

----
📚 Documentation preview 📚: https://volto--6231.org.readthedocs.build/

### CODE REVIEW

**Review by ichim-david — APPROVED** [review]

File: docs/source/configuration/multilingual.md [hunk 1]
```diff
@@ -28,22 +29,37 @@ config.settings = {
...
+
+## Docker backend configuration
+
+If you use the Docker image [`plone-backend`](https://github.com/plone/plone-backend), you can set its `LANGUAGE` environment variable, overriding the default of `en`, when you start it.
+
+You can either pass an environment variable into the make command to start the backend, or export an environment variable in your shell session and start the backend.
+
+```shell
+# pass method
+make backend-docker-start LANGUAGE=pt-br
```
Comment (wesleybl) [inline docs/source/configuration/multilingual.md:48]: I tried this command but the site was still created in English.

**Review by wesleybl — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -168,7 +170,7 @@ release-notes-copy-to-docs: ## Copy release notes into documentation
 
 .PHONY: backend-docker-start
 backend-docker-start: ## Starts a Docker-based backend for development
-	$(MAKE) -C "./packages/volto/" backend-docker-start
+	$(MAKE) -C "./packages/volto/" backend-docker-start -e $(LANGUAGE)
```
Comment (wesleybl) [inline Makefile:173]: I think the Volto makefile has to receive this variable:

https://github.com/plone/volto/blob/9e312ed0b1a977bcc42a2e6a7d1b9406ae7fb605/packages/volto/Makefile#L103

Comment (davisagli) [reply Makefile]: @wesleybl I made that adjustment and it works for me now if I do `LANGUAGE=pt-br make backend-docker-start`.

(I ran `docker volume rm volto-backend-data` first.)

File: docs/source/configuration/multilingual.md (hunk 1 above)
Comment (davisagli) [reply docs/source/configuration/multilingual.md:48]: LANGUAGE needs to be set as an environment variable in the shell, not passed to make as an argument:

```suggestion
LANGUAGE=pt-br make backend-docker-start
```

File: docs/source/configuration/multilingual.md [hunk 3]
```diff
@@ -28,22 +29,37 @@ config.settings = {
...
 ```{warning}
-The default language and the supported languages must match the one set in the Plone
-side, and those should be set using GenericSetup using your policy package, or
-manually via the Languages control panel, i.e. en for English, or pt-br for Portuguese (Brazil)
+The default language and the supported languages must match those set in the Plone backend.
+Those should be set either using GenericSetup using your policy package or manually via the {guilabel}`Languages` control panel.
+Examples include `en` for English, or `pt-br` for Portuguese (Brazil).
+```
+
+## Docker backend configuration
```
Comment (davisagli) [inline docs/source/configuration/multilingual.md:40]: @stevepiercy I think this is (arguably) the wrong place for this documentation. This chapter is about multilingual features, which means supporting more than one content language in the same Plone site. The LANGUAGE environment variable determines the first (and only) language for a new Plone site created by the plone-backend image, and it does not enable the multilingual features.

I think https://6.docs.plone.org/volto/contributing/developing-core.html#start-the-backend-and-volto is the most important place to mention the LANGUAGE option narrative docs.

File: Makefile (hunk 2 above)
Comment (wesleybl) [reply Makefile]: @davisagli Now it worked for me too!

File: docs/source/configuration/multilingual.md (hunk 1 above)
Comment (wesleybl) [reply docs/source/configuration/multilingual.md:48]: It works both ways.

**Review by wesleybl — APPROVED** [review]
LGTM

File: docs/source/configuration/multilingual.md (hunk 3 above)
Comment (stevepiercy) [reply docs/source/configuration/multilingual.md]: Agreed, thanks for finding a better home for it. I moved it there, and rearranged some bits and pieces to make it flow better and be easier to navigate.

**Comment by stevepiercy** [comment]
One final review for docs, please. Here's the preview:

https://volto--6231.org.readthedocs.build/contributing/developing-core.html

**Review by wesleybl — APPROVED** [review]
LGTM

**Review by ichim-david — APPROVED** [review]
Thanks Steve this is all nice to have documented.

File: docs/source/configuration/multilingual.md [hunk 4]
```diff
@@ -9,13 +9,14 @@ myst:
 
 # Multilingual
 
-Volto provide support for Plone's Multilingual feature. You need to install Multiligual
-support in Plone (`plone.app.multilingual` add-on), that comes available by default since
-Plone 5 and can be installed in Plone's control panel.
+Volto provides support for Plone's multilingual feature through the Plone core add-on `plone.app.multilingual`.
+It is included with Plone 5 and later.
+You can enable it through the Plone Classic UI administrative interface.
```
Comment (davisagli) [inline docs/source/configuration/multilingual.md:14]: Why did you change this to mention the Classic UI specifically? Add-ons can also be installed via the add-ons control panel in Volto.

File: docs/source/contributing/developing-core.md [hunk 5]
```diff
@@ -206,19 +216,38 @@ docker volume rm volto-backend-data
...
-    
-Browse to the backend running at http://localhost:8080.
 
-In the second session, start the frontend.
+
+(develop-volto-configure-backend-language-label)=
+
+#### Configure backend language
+
+If you use the Docker image [`plone-backend`](https://github.com/plone/plone-backend), you can set its `LANGUAGE` environment variable, overriding the default of `en`, when you start it.
```
Comment (davisagli) [inline docs/source/contributing/developing-core.md:225]: For what it's worth, this variable is only applied when the Plone site is created. If you start the backend multiple times without deleting the volume where it persists data, the LANGUAGE is only used the first time.

File: docs/source/configuration/multilingual.md (hunk 4 above)
Comment (stevepiercy) [reply docs/source/configuration/multilingual.md]: Done in b494e42aeecf689d111aac967beed0c6e0fc699d

File: docs/source/contributing/developing-core.md (hunk 5 above)
Comment (stevepiercy) [reply docs/source/contributing/developing-core.md:225]: Fixed in 83365fe5477a2f011df2dc6eb5828ed1233b141d. The bonus sentence may be obvious, but explicit yadda yadda...
